test/services/redeem_and_unstake/ProgressService/update.test.ts

```diff
@@ -0,0 +1,329 @@
+import sinon from 'sinon';
+import Web3 from 'web3';
+
+import { interacts } from '@openst/mosaic-contracts';
+
+import { AUXILIARY_GAS_PRICE, ORIGIN_GAS_PRICE } from '../../../../src/Constants';
+import Message from '../../../../src/models/Message';
+import GatewayRepository from '../../../../src/repositories/GatewayRepository';
+import { MessageStatus } from '../../../../src/repositories/MessageRepository';
+import ProgressService from '../../../../src/services/redeem_and_unstake/ProgressService';
+import Utils from '../../../../src/Utils';
+import SpyAssert from '../../../test_utils/SpyAssert';
+import StubData from '../../../test_utils/StubData';
+
+describe('ProgressService.update()', () => {
+  let coGatewaySpy: any;
+  let gatewaySpy: any;
+  let progressRedeemSpy: any;
+  let progressUnstakeSpy: any;
+  let utilsSpy: any;
+  let gatewayRepository: any;
+  let eip20CoGatewayMockObject: any;
+  let progressService: ProgressService;
+  let message: any;
+  const originWeb3 = new Web3(null);
+  const auxiliaryWeb3 = new Web3(null);
+  const originWorkerAddress = '0xA1e801AbF4288a38FfFEa3084C826B810c5d5294';
```

Comment:
any specific reasons for not having dummy values for origin and aux worker addresses ? As sendTx is mocked dummy values might work here

Reply:
I don't think we have strict requirement. Anyways I have updated the addresses.